### PR TITLE
Feature/svg gallery

### PR DESCRIPTION
**Pull request by leandrosveltestudios**
Add svg tabs for gallery and started integration

### CODE REVIEW

File: src/Gallery/Api/index.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ export type GalleryApi = {
   upload: (fileData: FormData) => Promise<GalleryItem>;
   save: (details: GalleryItem) => Promise<{ item: GalleryItem }>;
   load: (id: string) => Promise<{ item: GalleryItem }>;
+  //TODO : create api method to get svg string/object loadSvg: (id: string) => Promise<{ item: GalleryItem }>;
```
Comment (leandrosveltestudios) [inline src/Gallery/Api/index.ts:18]: Here I am not sure if we are going to send from API the actual SVG or a gallery object. If we load same as pictures, then  I think we should send the SVG directly.

File: src/Gallery/Images.tsx [hunk 2]
```diff
@@ -21,8 +21,8 @@ export const Images: FC<GalleryTab> = ({ items, upload, setSelected, selectedId
 
   return (
     <section className="whppt-gallery whppt-gallery__main-container">
-      <div className="whppt-gallery__grid">
-        <WhpptImageUploader uploadImage={uploadImage} />
+      <div className="whppt-gallery-grid whppt-gallery-grid--images">
```
Comment (leandrosveltestudios) [inline src/Gallery/Images.tsx:24]: Looking at the diff now on the PR I think `whppt-gallery-grid--images` is incorrect (is not a modifier), should be `__images`. Will change after feedback.

Comment (tjmkruger) [reply src/Gallery/Images.tsx]: agreed

File: src/Gallery/Images.tsx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import React, { FC } from 'react';
-import { WhpptImageUploader } from '../ui/components/ImageUploader';
-import { WhpptGalleryImage } from '../ui/components/GalleryImage';
-import { GalleryTab } from './GalleryTab';
+import { WhpptGalleryUploader } from './Components';
+import { WhpptGalleryImage } from './Components';
+import { GalleryTab } from './Components';
```
Comment (tjmkruger) [inline src/Gallery/Images.tsx:4]: These 3 can be a single import

File: src/Gallery/index.tsx [hunk 4]
```diff
@@ -32,21 +34,20 @@ export const Gallery: FC<{ onUse?: (image: GalleryItem) => void }> = ({ onUse })
...
     const type = settingsPanel.activeTab as GalleryFileType;
-
     const search = api.gallery
       .search({ domainId: domain._id, page: 1, size: 10, type, tags, filter })
       .then(({ items }: { items: GalleryItem[] }) => setItems(items))
       .catch(error => setError(error.message || error));
 
     return toast.promise(search, {
-      error: 'Image upload failed 🤯',
+      error: 'Image search failed 🤯',
```
Comment (tjmkruger) [inline src/Gallery/index.tsx:43]: I guess this search could be for more than just images. Could we make it "Gallery search failed"?

File: src/styles/gallery.scss [hunk 5]
```diff
@@ -65,6 +73,33 @@
     @apply whitespace-nowrap overflow-hidden text-ellipsis;
   }
 
+  &__svg {
+    @apply h-galleryItem w-gallerySvg cursor-pointer;
+    $size: 50px;
```
Comment (tjmkruger) [inline src/styles/gallery.scss]: This should be using the checkers mixin now. Note to check

File: src/Gallery/Api/index.ts [hunk 6]
```diff
@@ -15,6 +15,8 @@ export type GalleryApi = {
...
```
Comment (tjmkruger) [inline src/Gallery/Api/index.ts:18]: Can remove the comment

**Pull request merged**